data_dir/cdc_profile.yaml

```diff
@@ -0,0 +1,75 @@
+keyspace: cdc_test
+
+keyspace_definition: |
+
+  CREATE KEYSPACE cdc_test WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '3'}  AND durable_writes = true;
+
+table: test_table
+
+table_definition: |
+
+  CREATE TABLE cdc_test.test_table (
+    primaryrepositoryid text PRIMARY KEY,
```

Comment:
Where this table definitions are coming from ? can we name the columns a bit clearer do investigations would be easier ?

Reply:
i think better to leave only fields with unique types may be all supported types

Reply:
Had fixed.